indico/modules/events/abstracts/__init__.py

```diff
@@ -141,3 +142,10 @@ def _reviewing_area_visible(conf):
     yield MenuEntryData(title=_("Reviewing Area"), name='user_tracks', endpoint='abstracts.display_reviewable_tracks',
                         position=0, parent='call_for_abstracts',
                         visible=_reviewing_area_visible)
+
+
+@template_hook('conference-home-info')
+def _inject_cfa_announcement(event, **kwargs):
+    event = event.as_event
+    if event.cfa.is_open or (session.user and event.cfa.can_submit_abstracts(session.user)):
```

Comment:
check whether the feature is enabled first. `can_submit_abstracts` always triggers a DB query to get the list of authorized users if the cfa is not open